humanize/plural.go

```diff
@@ -5,13 +5,15 @@ import (
 	"strconv"
 )
 
-func PluralWord(quantity int, singular, plural string) string {
+// Plural returns the quantity then either the `singular` (fox) or `plural`
+// (foxes) depending on that given `quantity`
+// e.g. `Plural(2, "fox", "foxes")` returns "2 foxes"
+func Plural(quantity int, singular, plural string) string {
```

Comment:
Just a thought, losing the separation of `Plural` and `PluralWord` is a shame, because there's now no way of doing, for example, "no foxes" or "three foxes". I guess what i meant was that it wasn't clear what `Plural` was doing (since it's a noun). *Pluralize* would be better. `PluralWord` made total sense.

Reply:
That said, happy to crack on and address this when it's needed. From a general style point of view, we should try and use verbs at the start of functions I reckon.

Reply:
Let's refactor it when we need it. Good idea about verbs.